.gitignore

```diff
@@ -15,4 +16,4 @@ Thumbs.db
 .idea
 .vscode
 *.sublime-project
-*.sublime-workspace
\ No newline at end of file
+*.sublime-workspace
```

Comment:
Sorry I'm not sure what you want excluded here, the line was already there my editor just inserted a newline character. I'm not even using sublime